config/locales/en.yml

```diff
@@ -1849,43 +1848,17 @@ en:
   sessions:
     new:
       title: "Login"
```

Comment:
Login is inconsistently spelled "Log in"  or "Login" throughout this PR.

Reply:
English is not my 1st language, so I need some guidance here: it is not clear to me when "log in" or "login" should be used.

OSM website was not consistent to start with: Here are the counts as reported by `grep -i "log in" config/locales/en.yml` and `grep -i "login" config/locales/en.yml` on master and this PR.

```
|        | master | this PR |
+--------+--------+---------+
| log in |    7   |    8    |
| login  |   29   |   30    |

Should I go ahead and replace all instances of "log in" with "login"?
```

Reply:
Well wiktionary suggests that it is "login" when used as a noun, and "log in" when used as a verb. In fact it lists "login" as a verb as a mis-spelling of "log in" and refers to http://notaverb.com/login on the subject...

Reply:
> English is not my 1st language

I don't think that would actually help as we are very very inconsistent in general (outside of OSM), there are at least login, log in, log-in, and log-on in use (not to mention that sign-in would work to, but that might get confused with sign-up). IMHO @gravitystorm  and @tomhughes should simply decide on one variant.

Reply:
One of them is a verb, another is a noun. For buttons you'd use the "Log in" verb, but looks like we currently don't.

Reply:
> One of them is a verb, another is a noun. For buttons you'd use the "Log in" verb, but looks like we currently don't.

Thanks, I will review my modifications having this suggestion in mind.